docs/standard-library.rst

```diff
@@ -58,6 +58,9 @@ Processors
 :func:`~structlog.stdlib.add_logger_name`:
    Adds the name of the logger to the event dictionary under the key ``logger``.
 
+:func:`~structlog.stdlib.add_log_level_number`:
```

Comment:
Same as above, maybe explain a bit more what a log level number is? Someone reading this who doesn’t know what a log level number is, will have not clue what this is support to be.